src/example/heat_equation_sphere/heat_equation_sphere.jl

```diff
@@ -0,0 +1,224 @@
+        θ = PhysicalFunction(x -> cart2sphere(x)[2])
+        θ_centers = var_on_centers(θ, mesh)
+        θ_vertices = var_on_vertices(θ, mesh)
+
+        ϕ = PhysicalFunction(x -> cart2sphere(x)[3])
+        ϕ_centers = var_on_centers(ϕ, mesh)
+        ϕ_vertices = var_on_vertices(ϕ, mesh)
+
+        ν = Bcube.CellNormal(mesh)
+        ν_centers = transpose(var_on_centers(ν, mesh))
+        ν_vertices = transpose(var_on_vertices(ν, mesh))
+
+        new(
+            basename,
+            0,
+            mesh,
+            θ,
+            θ_centers,
+            θ_vertices,
+            ϕ,
+            ϕ_centers,
+            ϕ_vertices,
+            ν,
+            ν_centers,
+            ν_vertices,
+        )
+    end
+end
+
+function append_vtk(vtk, u::Bcube.AbstractFEFunction, t)
+    values_vertices = var_on_vertices(u, vtk.mesh)
+    values_centers = var_on_centers(u, vtk.mesh)
+
+    ## Write
+    Bcube.write_vtk(
+        vtk.basename,
+        vtk.ite,
+        t,
+        vtk.mesh,
+        Dict(
+            "u_centers" => (values_centers, VTKCellData()),
+            "u_vertices" => (values_vertices, VTKPointData()),
+            "θ_centers" => (vtk.θ_centers, VTKCellData()),
+            "θ_vertices" => (vtk.θ_vertices, VTKPointData()),
+            "ϕ_centers" => (vtk.ϕ_centers, VTKCellData()),
+            "ϕ_vertices" => (vtk.ϕ_vertices, VTKPointData()),
+            "ν_centers" => (vtk.ν_centers, VTKCellData()),
+            "ν_vertices" => (vtk.ν_vertices, VTKPointData()),
+        ),
+        ;
+        append = vtk.ite > 0,
+    )
+
+    ## Update counter
+    vtk.ite += 1
+end
+
+function run(;
+    α = 1.0 / 42,
+    degree = 1,
+    tfinal = 1,
+    nout = 100,
+    lc = 1e-1,
+    CFL = 0.5,
+    vtk_output = true,
+)
+
+    # Settings
+    out_dir = joinpath(@__DIR__, "../../../myout/heat_eqn_sphere")
+    mkpath(out_dir)
+
+    # Mesh
+    mesh_path = joinpath(out_dir, "mesh.msh")
+    Bcube.gen_sphere_mesh(mesh_path; radius = 1.0, lc = lc)
+    mesh = read_msh(mesh_path)
+    rng = MersenneTwister(0)
+    R = rotMat(rand(rng, 3)...)
+    transform!(mesh, x -> R * x) # rotate to avoid being "aligned" with an axis
+
+    # Domain
+    dΩ = Measure(CellDomain(mesh), 2 * degree + 1)
+
+    # Prepare output
+    vtk = VtkHandler(joinpath(out_dir, "result_d$(degree)"), mesh)
+
+    # Discretization
+    U = TrialFESpace(FunctionSpace(:Lagrange, degree), mesh)
+    V = TestFESpace(U)
+    u = FEFunction(U)
+
+    # Display infos
+    println("degree = $degree, CFL = $CFL, lc = $lc, ndofs = $(get_ndofs(U))")
+
+    # Init
+    u0 = PhysicalFunction(
+        x -> begin
+            _r, _θ, _ϕ = cart2sphere(x)
+            return sphevaluate(_θ, _ϕ, 6, 0) + √(14 / 11) * sphevaluate(_θ, _ϕ, 6, 5)
+        end,
+    )
+    projection_l2!(u, u0, mesh)
+
+    # Bilinear forms
+    m(u, v) = ∫(u ⋅ v)dΩ
+    a(u, v) = ∫(∇ₛ(u) ⋅ ∇ₛ(v))dΩ
+
+    # Assemble
+    M = assemble_bilinear(m, U, V)
+    K = assemble_bilinear(a, U, V)
```

Comment:
add `factorize` step ?

Reply:
no need since it's a steady resolution, so the inversion will occur only once

Reply:
What do you mean by steady resolution?

Reply:
Well, you're right it's not steady. I mean there is only one time step, i.e one linear system inversion. So it's not necessary to factorize.